tool/gen_mackerel_plugin.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+use 5.014;
+use warnings;
+use utf8;
+use autodie;
+use JSON::PP;
+
+my $json = do {
+    local $/;
+    open my $fh, '<', 'packaging/config.json';
+    <$fh>
+};
+
+my @plugins = sort @{ decode_json($json)->{plugins}};
+
+sub resolve_package {
+    my $plug = shift;
+    $plug =~ s/-//g;
+    "mp$plug";
+}
+
+my $imports = "";
+my $case = "";
+my $plugs = "";
+for my $plug (@plugins) {
+    my $pkg = "mp$plug";
+       $pkg =~ s/-//g;
+    $imports .= sprintf qq[\t"github.com/mackerelio/mackerel-agent-plugins/mackerel-plugin-%s/lib"\n], $plug;
+    $case .= sprintf qq[\tcase "%s":\n\t\t%s.Do()\n], $plug, $pkg;
+    $plugs .= sprintf qq[\t"%s",\n], $plug;
+}
+
+my $mackerel_plugin_gen = qq!// Code generated by "tools/gen_mackerel_plugin.pl"; DO NOT EDIT
+package main
+
+import (
+	"fmt"
+
+$imports)
+
+func runPlugin(plug string) error {
+	switch plug {
+${case}\tdefault:
+		return fmt.Errorf("unknown plugin: %q", plug)
+	}
+	return nil
+}
+
+var plugins = []string{
```

Comment:
[nit] You can use `[...]string{...}` to make plugins var array, not slice.

https://golang.org/ref/spec#Composite_literals